WooCommerce/Classes/ViewModels/Order Details/OrderStatusListViewModel.swift

```diff
@@ -27,19 +29,27 @@ final class OrderStatusListViewModel {
     ///
     private(set) var shouldEnableApplyButton: Bool = false
 
+    /// Whether to automatically confirm the order status when it is selected.
+    ///
+    /// Defaults to `false`.
+    ///
+    let isSelectionAutoConfirmed: Bool
```

Comment:
nit: I found this name a little bit funny, do you think something like `autoConfirmSelection` or `confirmsSelectionAutomatically` will be easier to read/reason?